common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockIdUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.network.shuffle;
+
+import org.apache.spark.network.shuffle.checksum.ShuffleChecksumHelper;
+
+public class BlockIdUtils {
+  public static final String SHUFFLE_BLOCK_PREFIX = "shuffle_";
+  public static final String SHUFFLE_CHUNK_PREFIX = "shuffleChunk_";
+  public static final String RDD_BLOCK_PREFIX = "rdd_";
```

Comment:
We already have the pattern of blocks in [object BlockId](https://github.com/apache/spark/blob/336ca8c12a163d31519a491a28d26ff091626986/core/src/main/scala/org/apache/spark/storage/BlockId.scala#L203). Can we reuse this?